src/autocomplete.js

```diff
@@ -27,6 +27,7 @@ var preventParentScroll = require("./lib/scroll").preventParentScroll;
  * @property {string} [docHTML] - an HTML string that would be displayed as an additional popup
  * @property {string} [docText] - a plain text that would be displayed as an additional popup. If `docHTML` exists,
  * it would be used instead of `docText`.
+ * @property {Function} [clickCallback] - a callback function which can be associated with a popup item. If present text value won't be used after clicking the popup item.
```

Comment:
nitpick: looking at other [handlers](https://github.com/ajaxorg/ace/blob/fd61361d8307cc3d7307a16e7d96b228ed47d173/types/ace-modules.d.ts#L1607) we have for similar functionality, could it make sense to also call this something like `on<xyz>`, so maybe `onClick` or `onItemClick`